src/module.c

```diff
@@ -2918,6 +2918,9 @@ int RM_ReplyWithCallReply(RedisModuleCtx *ctx, RedisModuleCallReply *reply) {
     size_t proto_len;
     const char *proto = callReplyGetProto(reply, &proto_len);
     addReplyProto(c, proto, proto_len);
+    list *errors = callReplyDeferredErrorList(reply);
```

Comment:
```suggestion
================================================================================
    /* Propagate the error list from that reply to the other client, to do some
     * post error reply handling, like statistics.
     * Note that if the original reply had an array with errors, and the module
     * replied with just a portion of the original reply, and not the entire
     * reply, the errors are currently not propagated and the errors stats
     * will not get propagated. */
    list *errors = callReplyDeferredErrorList(reply);
```

Reply:
@yossigo Meir and I think this compromise is ok (not to complicate the code with parsing the partial reply).
please ack.

Reply:
Makes perfect sense, I think this is too esoteric to deal with. I think any practical use case involves either custom replies by the module or pass-through of the entire call reply.